test/common.py

```diff
@@ -719,10 +737,13 @@ def parameterize(func, parameters):
   if prev:
     # If we're parameterizing 2nd time, construct a cartesian product for various combinations.
     func._parameterize = {
-      '_'.join(filter(None, [k1, k2])): v1 + v2 for (k1, v1), (k2, v2) in itertools.product(prev.items(), parameters.items())
+      '_'.join(filter(None, [k1, k2])): v2 + v1 for (k1, v1), (k2, v2) in itertools.product(prev.items(), parameters.items())
```

Comment:
The rational here is that the outermost wrapper thats the leftmost argument.

So `foo_proxied_wasmfs` takes wasmfs as arg0 and proxied as arg1.